shared/tinyusb/mp_usbd_cdc.c

```diff
@@ -149,13 +149,7 @@ static void usbd_cdc_run_bootloader_task(mp_sched_node_t *node) {
 }
 #endif
 
-void
-#if MICROPY_HW_USB_EXTERNAL_TINYUSB
-mp_usbd_line_state_cb
-#else
-tud_cdc_line_state_cb
-#endif
-    (uint8_t itf, bool dtr, bool rts) {
+void tud_cdc_line_state_cb(uint8_t itf, bool dtr, bool rts) {
```

Comment:
This change (in the first commit here) needs a corresponding change in `mp_usbd_cdc.h` to remove the public declaration of `mp_usbd_line_state_cb()`.

Reply:
Ah yes I only just realised that earlier today myself! I'll fix it and rebase asap thanks.